tests/Custom/WebDriverTest.php

```diff
@@ -72,4 +76,32 @@ public function testClassicDriverCanProvideBrowserName(): void
             $this->driver->getBrowserName()
         );
     }
+
+    public function testThatDriverCatchesUnexpectedAttributeValueType(): void
+    {
+        $mockWebDriver = $this->createMock(RemoteWebDriver::class);
+        $mockWebDriverOptions = $this->createMock(WebDriverOptions::class);
+        $mockWebDriverTimeouts = $this->createMock(WebDriverTimeouts::class);
+        $mockElement = $this->createMock(RemoteWebElement::class);
+        $mockWebDriver->method('manage')->willReturn($mockWebDriverOptions);
+        $mockWebDriverOptions->method('timeouts')->willReturn($mockWebDriverTimeouts);
```

Comment:
It's a huge code block and reading becomes even more difficult because the variable declaration order and usage order don't match.

I'm proposing to rearrange them like so:

```php
$mockWebDriverTimeouts = $this->createMock(WebDriverTimeouts::class);
$mockWebDriverOptions = $this->createMock(WebDriverOptions::class);
$mockWebDriverOptions->method('timeouts')->willReturn($mockWebDriverTimeouts);

$mockElement = $this->createMock(RemoteWebElement::class);

$mockWebDriver = $this->createMock(RemoteWebDriver::class);
$mockWebDriver->method('manage')->willReturn($mockWebDriverOptions);
```

Reply:
fixed in merge